k8s/us-central1/volumes.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: PersistentVolume
```

Comment:
where in this PR do we point the volume to the PD? (Also: we still use PD?)

Could you file a bug to write a guide similar to [this](https://docs.google.com/document/d/19NEyXYR7HN12gCJ0uOg6TBe4GVW_3rq6xbBjFIJx7_A/edit#heading=h.jz0jr8sninst) on how a user would create a dataset storage vehicle to interact with this new volume stuff

Reply:
The `nfs` field below has the IP and fileshare name that we use for this cluster. I will add a note to b/152724749 to include details on how to add datasets to the Filestore instance. None of the tests using `PersistentVolumeClaim`s are using PDs anymore.